test/spoom/sorbet/sigils_test.rb

```diff
@@ -0,0 +1,229 @@
+# typed: true
+# frozen_string_literal: true
+
+require "test_helper"
+
+module Spoom
+  module Sorbet
+    class SigilsTest < Minitest::Test
+      def test_sigil_nonempty
+        sigil = Sigils.sigil("false")
+        assert_equal("# typed: false", sigil)
+      end
+
+      def test_sigil_empty
+        sigil = Sigils.sigil("")
+        assert_equal("# typed: ", sigil)
+      end
+
+      def test_valid_strictness_ignore
+        content  = <<~STR
```

Comment:
This feels a bit redundant.

Could we do something like:

```ruby
["ignore", "false", "true", "strict", "strong", "   strong   "].each do |strictness|
  content  = <<~STR
    # typed: #{strictness}
    class A; end
  STR
  assert(Sigils.valid_strictness?(content))
end
```

And another one for when it should return false:

```ruby
["", "FALSE", "foo"].each do |strictness|
  content  = <<~STR
    # typed: #{strictness}
    class A; end
  STR
  refute(Sigils.valid_strictness?(content))
end
```